### DISCUSSION TITLE
Prompt  for accurate vehicle appraisal florida  chatgpt

### DISCUSSION THREAD

**Discussion by Trailblazerrtyuu** [community/community#148232] [Programming Help] [answered]
### Select Topic Area

General

### GitHub Feature Area

Command Palette

### Body

from docx import Document

# Create Outline Document
outline_doc = Document()
outline_doc.add_heading("Comprehensive Auto Appraisal Report Outline", level=1)

sections = [
    ("1. Appraisal Summary", [
        "Overview: Brief description of the appraisal purpose and key findings.",
        "Vehicle Summary: Make, model, year, VIN, and ACV estimate.",
        "Legal Compliance: Statement of adherence to Florida statutes and USPAP standards."
    ]),
    ("2. Appraiser Information", [
        "Name: [Appraiser Name]",
        "License/Certification Details: [License Number, Certification Agency]",
        "Professional Credentials: Brief summary of experience and qualifications.",
        "Contact Information: [Phone Number, Email, Office Address]"
    ]),
    ("3. Owner and Vehicle Information", [
        "Owner Details:",
        "- Name: [Owner Name]",
        "- Contact Information: [Phone, Email]",
        "- Lienholders (if any): [Lienholder Name & Contact Information]",
        "Vehicle Details:",
        "- Make and Model: [Make, Model]",
        "- Year: [Year]",
        "- VIN: [VIN]",
        "- Mileage: [Mileage (actual, not actual)]",
        "- Registration Status: [Active, Expired, etc.]",
        "- Title Status: [Clear, Salvage, Branded]"
    ]),
    ("4. Vehicle Condition", [
        "Exterior:",
        "- Description: Detailed notes on paint, body, trim, and glass.",
        "- Photos: Include images of any damages or unique aspects.",
        "Interior:",
        "- Description: Condition of upholstery, dashboard, electronics, and controls.",
        "- Photos: Document key areas like seats, carpets, and dashboard.",
        "Mechanical:",
        "- Condition Assessment: Engine, transmission, suspension, brakes, and tires.",
        "- Service Records: Refer to available records, if applicable.",
        "Grading Scale: Provide overall condition grade (e.g., Excellent, Good, Fair, Poor)."
    ]),
    ("5. Features and Optional Equipment", [
        "- Standard Features: List of default features for the model.",
        "- Optional Equipment: Any added features (e.g., premium audio, sunroof).",
        "- Upgrades: Note aftermarket additions or modifications.",
        "- Impact on Valuation: Explanation of how features influence ACV."
    ]),
    ("6. Vehicle History", [
        "- Accident History: Records from databases like Carfax or AutoCheck.",
        "- Service History: Summary of maintenance records, if available.",
        "- Previous Owners: Number of owners and length of ownership.",
        "- Notable Events: Salvage, flood, or theft records."
    ]),
    ("7. Market Analysis", [
        "Comparable Vehicle Sales:",
        "| Make/Model/Year | Mileage | Condition | Sale Price | Source |",
        "|------------------|---------|-----------|------------|--------|",
        "| [Example 1]      | [XX,XXX] | [Condition] | [$Amount] | [Source] |",
        "Market Trends: Current trends affecting similar vehicles (e.g., seasonal demand, supply chain impacts).",
        "Methodology: Explanation of valuation method: adjustments for mileage, condition, or features."
    ]),
    ("8. Valuation", [
        "- Valuation: $[Amount]",
        "- Justification: Supporting data and rationale, including adjustments made for condition or features.",
        "- Additional Considerations: Taxes, registration fees, or other costs (if required by purpose)."
    ]),
    ("9. Certification and Disclaimer", [
        "Certification: 'I, [Appraiser Name], certify that this appraisal was conducted in accordance with Florida law, insurance regulations, and professional standards.'",
        "Disclaimer:",
        "- 'This report is intended solely for the stated purpose and should not be used for other purposes.'",
        "- 'The valuation reflects the market conditions and data available at the time of appraisal.'"
    ]),
    ("10. Supporting Evidence", [
        "- Photos: Attach clear images of:",
        "  - Vehicle exterior (all sides, damages).",
        "  - Interior (seats, dashboard, electronics).",
        "  - Mechanical components (engine, undercarriage).",
        "- Source Data: Copies or references to:",
        "  - Comparable vehicle sales reports.",
        "  - Service and accident history reports.",
        "  - Valuation methodology sources (e.g., NADA, KBB)."
    ]),
    ("11. Dispute Resolution", [
        "- Process: Outline the steps for disputing the ACV (e.g., mediation, arbitration).",
        "- References: Florida statutes (e.g., Fla. Stat. § 627.7015) for mediation or arbitration options."
    ]),
    ("12. Notes and Additional Comments", [
        "- Include any additional observations or notes that may not fit into other sections.",
        "- Relevant insights or information that may aid in understanding the appraisal."
    ])
]

for section, details in sections:
    outline_doc.add_heading(section, level=2)
    for detail in details:
        outline_doc.add_paragraph(detail)

# Save the outline document
outline_path = "/mnt/data/Auto_Appraisal_Report_Outline.docx"
outline_doc.save(outline_path)

# Create Prompts Document
prompts_doc = Document()
prompts_doc.add_heading("Comprehensive Auto Appraisal Prompts", level=1)

prompts = [
    "Summarize the purpose, key findings, and ACV of the vehicle, including adherence to Florida statutes and USPAP standards. Ensure clarity and professionalism.",
    "Provide the appraiser’s name, license or certification details, professional credentials, and contact information for the report. Ensure compliance with Florida disclosure requirements.",
    "Include the owner's name, contact details, lienholder information (if any), and complete vehicle details such as make, model, year, VIN, mileage, registration status, and title condition.",
    "Assess and describe the vehicle's exterior, interior, and mechanical condition. Use a grading scale (Excellent, Good, Fair, Poor), and include placeholders for photographic evidence if not provided.",
    "List the vehicle’s standard features, optional equipment, and any aftermarket upgrades. Explain how these features impact the ACV, if applicable.",
    "Summarize the vehicle’s accident history, service history, ownership details, and any notable events like salvage or flood records. Reference databases like Carfax or AutoCheck.",
    "Provide a market analysis including data on comparable vehicle sales, market trends, and valuation methodology. Include a table of comparables and cite reliable sources such as Kelley Blue Book or NADA.",
    "Calculate and specify the vehicle’s ACV with a clear justification. Include adjustments for condition, features, and market trends, and address any additional costs such as taxes or fees.",
    "Draft the certification and disclaimer for the report, ensuring it complies with Florida statutes. Certify the appraisal's professionalism and include limitations of scope.",
    "List required supporting evidence, such as vehicle photos, source data for comparables, and service or accident history reports. Ensure all references are relevant and verifiable.",
    "Outline a clear process for disputing the ACV, referencing Florida statutes like Fla. Stat. § 627.7015 for mediation or arbitration options.",
    "Add any observations or notes that do not fit into other sections. Include relevant insights or information that may aid in understanding the appraisal."
]

for idx, prompt in enumerate(prompts, start=1):
    prompts_doc.add_heading(f"Prompt {idx}", level=2)
    prompts_doc.add_paragraph(prompt)

# Save the prompts document
prompts_path = "/mnt/data/Auto_Appraisal_Prompts.docx"
prompts_doc.save(prompts_path)

(prompts_path, outline_path)

**Comment by queenofcorgis** [comment, marked as answer]
Hey there!  👋 

Thanks for posting in the GitHub Community, @Trailblazerrtyuu ! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category. The Accessibility category is a place for our community to discuss and provide feedback on the digital accessibility of GitHub products. Digital accessibility means that GitHub tools, and technologies, are designed and developed so that people with disabilities can use them.

We’ve moved your post to our Programming Help 🧑‍💻 category, which is more appropriate for this type of discussion.

Please review our [guidelines](https://github.com/orgs/community/discussions/51384) about the Programming Help category for more information.
